cookiejs/cookiejs.d.ts

```diff
@@ -2,20 +2,46 @@
 // Project: https://github.com/js-coder/cookie.js
 // Definitions by: Boltmade <https://github.com/Boltmade>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-
+/**
+ * Shortcut for cookie.get()
+ */
 declare function cookie(key : string, fallback?: string) : string;
 declare function cookie(keys : string[], fallback?: string) : string;
 
 declare namespace cookie {
+  /**
+   * Create a cookie. The value will automatically be escaped.
+   */
   export function set(key : string, value : string, options? : any) : void;
+  /**
+   * Set several cookies at once
+   */
   export function set(obj : any, options? : any) : void;
+  /**
+   * Remove cookies
+   */
   export function remove(key : string) : void;
   export function remove(keys : string[]) : void;
   export function remove(...args : string[]) : void;
+  /**
+   * Remove all cookies
+   */
   export function empty() : void;
+  /**
+   * Retrieve the value of the cookie
+   */
   export function get(key : string, fallback?: string) : string;
-  export function get(keys : string[], fallback?: string) : string;
```

Comment:
https://github.com/florian/cookie.js#cookieget
please use `string | string[]` instead of `string[]`.